taskvine/src/bindings/python3/taskvine.binding.py

```diff
@@ -59,7 +59,23 @@ def __del__(self):
             # ignore exceptions, in case task has been already collected
             pass
 
+    ##
+    # Return the contents of a file object as a string.
+    # Typically used to return the contents of an output buffer.
+    #
+    # @param self       A file object.
+    
+    def contents(self):
+        return vine_file_contents(self._file)
 
+    ##
+    # Return the length of a file object.
+    #
+    # @param self       A file object.
+    
+    def length(self):
```

Comment:
Suggest the more pythonic `def __len__(self)`. Code then calls `len(File(...))`

Reply:
That's a good idea.  I was tempted to put in `__str__` instead of `contents`, but wasn't ready to decide if the default string should be the file contents, the source, the remote name, or...